SensusUI/PromptForInputsPage.cs

```diff
@@ -121,36 +142,49 @@ public PromptForInputsPage(InputGroup inputGroup, int stepNumber, int totalSteps
             if (_displayedInputCount > 0)
                 contentLayout.Children.Add(new BoxView { Color = Color.Transparent, HeightRequest = inputSeparatorHeight });
 
-            if (anyRequired)
-                contentLayout.Children.Add(new Label
-                    {
-                        Text = "* Required Field",
-                        FontSize = 15,
-                        TextColor = Color.Red,
-                        HorizontalOptions = LayoutOptions.FillAndExpand
-                    });
-
             StackLayout navigationStack = new StackLayout
             {
-                Orientation = StackOrientation.Horizontal,
+                Orientation = StackOrientation.Vertical,
                 HorizontalOptions = LayoutOptions.FillAndExpand
             };
 
+            StackLayout previousNextStack = new StackLayout
+                {
+                    Orientation = StackOrientation.Horizontal,
+                    HorizontalOptions = LayoutOptions.FillAndExpand,
+//                    Padding = new Thickness(0, 0, 0, 15)
+                };
+
+            StackLayout previousStack = new StackLayout
+                {
+                    Orientation = StackOrientation.Horizontal,
+                    HorizontalOptions = LayoutOptions.FillAndExpand
+                };
+
+            StackLayout nextStack = new StackLayout
+                {
+                    Orientation = StackOrientation.Horizontal,
+                    HorizontalOptions = LayoutOptions.FillAndExpand
+                };
+
+            previousNextStack.Children.Add(previousStack);
+            previousNextStack.Children.Add(nextStack);
+
```

Comment:
You don't need to put the previous and next buttons in stacks, since they're by themselves anyway. Also, since there will now always be a previous button, there is no need for the `if`-check on `stepNumber` below. Initialize the entire `previousNextStack` using a single statement (only 1 semi-colon), and use the ternary conditional to set the color of the previous button depending on the step number.